agents/agents-mcp/src/jvmMain/kotlin/ai/koog/agents/mcp/McpToolDefinitionParser.kt

```diff
@@ -91,6 +91,25 @@ public object DefaultMcpToolDescriptorParser : McpToolDescriptorParser {
                     return parseParameterType(nonNullType, depth + 1)
                 }
             }
+
+            /**
+             * Special case for enum string types.
+             * Schema example:
+             * {
+             *   "enumParam": {
+             *     "enum": [
+             *       "value1",
+             *       "value2"
+             *     ],
+             *     "title": "Enum string parameter"
+             *   }
+             * }
+             */
+            val enum = element["enum"]?.jsonArray
```

Comment:
Sorry, maybe I don’t quite understand. But is this example actually valid? Because usually an enum looks like this:

```json
{
  "type": "string",
  "enum": ["red", "green", "blue"]
}
```

Reply:
As far as I know, title is an optional field, but type is required